langs/ru.ini

```diff
@@ -1,9 +1,19 @@
 name="Русский"
 dir="ltr"
-email="E-mail"
+missing="Извините, здесь нет перевода"
+readmore="Читать дальше"
+languages="языки"
+section="фрагмент"
+shortcuts="сокращения"
+legal="правовые информации"
+email="адрес электронной почты"
 fax="Факс"
 telephone="Телефон"
 bus="Автобус"
 subway="Метро"
+skip="перейти к содержанию"
+newsletter="подписаться на рассылку"
+subscribe="подписаться"
+map="план города"
```

Comment:
Did you do these manually? The easiest way is to use [transifex-client](https://pypi.org/project/transifex-client/)

Reply:
Thanks for the tip!